src/main/resources/layouts/settings.fxml

```diff
@@ -283,6 +283,35 @@
                      </children>
                   </Group>
                </children>
+            </VBox>   <VBox styleClass="menuBorder">
+               <children>
+                  <Group>
+                     <children>
+                        <VBox>
+                           <children>
+                              <HBox spacing="10.0">
+                                 <children>
+                                    <Label text="Note Reminder">
+                                       <font>
+                                          <Font name="Open Sans Bold" size="12.0" />
+                                       </font>
+                                    </Label>
+                                 </children>
+                              </HBox>
+                              <HBox spacing="10.0">
+                                 <children>
+                                    <CheckBox fx:id="emptyNoteReminderCheckBoxIsWork" mnemonicParsing="false" text="Ask for notes when switching from project marked as work (if empty)" wrapText="true" HBox.hgrow="NEVER">
```

Comment:
the new option name (and variable names) are not so clear to be.
If the option is not selected KeepTime should ask for a note for each project (if note reminder is on). If true, only for the ones which are work.

e.g. renaming "remindIfNotesAreEmptyIsWork" to "remindIfNotesAreEmptyOnlyForIsWork" would make it a little clearer what it means. maybe there is even better name.